scripts/functions/requirements/arch

```diff
@@ -64,12 +69,15 @@ requirements_arch_define()
       requirements_check nodejs
       ;;
     (rbx*head|rubinius*head)
+      undesired_check clang35 llvm35 llvm35-libs
       requirements_check clang llvm llvm-libs patch curl zlib readline autoconf automake diffutils make libtool bison git
       ;;
     (rbx-3*|rubinius-3*)
+      undesired_check clang35 llvm35 llvm35-libs
```

Comment:
Looking at #3497 seems that *35 versions are incompatible with current versions, so I guess we need to dump all of it.